aquarius/app/assets.py

```diff
@@ -461,6 +465,84 @@ def update(did):
         return f'Some error: {str(err)}', 500
 
 
+@assets.route('/ddo/transferownership/<did>', methods=['PUT'])
+def transferownership(did):
+    """Update DDO of an existing asset
+    ---
+    tags:
+      - ddo
+    consumes:
+      - application/json
+    parameters:
+      - name: did
+        in: path
+        description: DID of the asset.
+        required: true
+        type: string
+        example: "did:op:d007b84d6f874cbf868177898f2353f7adfc824c9f9843d8b9ee60596db3b9f0"
+      - in: body
+        name: body
+        required: true
+        schema:
+          type: object
+          required:
+            - newowner
+            - updated
+            - signature
+          properties:
+            "signature":
+              description: Signature using updated field to verify that the consumer has rights to update onwership
+              type: string
+              example: "0x42e940108a430b91796341e29001319b2b2c4743156cdbe0e17afdae82b4cf9a7e1b4e641cd57d8f087ab6432cc9e53989f3ce121b6897fa3f594e9753c4ea331b"
+            "updated":
+              description: Last update field of the DDO
+              type: string
+              example: "2020-01-01T00:00:00Z"
+            "newowner":
+              description: The new owner ethereum address.
+              type: string
+              example: "0x858048e3Ebdd3754e14F63d1185F8252eF142393"
+    responses:
+      200:
+        description: Asset successfully transfered.
+      400:
+        description: One of the required attributes is missing.
+      404:
+        description: Invalid asset data.
+      500:
+        description: Error
+    """
+    data = request.json
+    required_attributes = [
+        'signature',
+        'updated',
+        'newowner'
+    ]
+    msg, status = check_required_attributes(required_attributes, data, 'transferownership')
+    if msg:
+        return msg, status
+    if web3.isAddress(data['newowner'])==False:
```

Comment:
if statement can be simplified: `if not web3.isAddress(data['newowner']):`

Reply:
fixed